src/PlayingCards.tsx

```diff
@@ -49,14 +52,19 @@ function PlayingCard(props: CardProps) {
       isDragging: !!monitor.isDragging(),
     }),
   });
-
   return (
     <div
       className={classNames("playingCard")}
       onClick={toggleSide}
       style={{
-        opacity: isDragging ? 0.7 : 1,
-        transform: isOver ? "translateY(20px)" : "",
+        opacity: isDragging ? 0.5 : 1,
+        transform: isOver
+          ? directionOfDrag === Direction.Left
+            ? "translateX(30px)"
+            : directionOfDrag === Direction.Right
+            ? "translateX(-30px)"
+            : ""
+          : "",
```

Comment:
same as the comment above: its better to use classnames

Reply:
is there a cleaner way to do nested if else?

Reply:
My advice is to abandon if/else here, and use className like this:

```
className={classnames("playing-card", {
  hovered: isOver,
  directionLeft: directionOfDrag === Direction.Left,
  directionRight: directionOfDrag === Direction.Right
})}
```

then in your CSS file:

```
.playing-card.hovered.directionLeft {
  transform: translateX(30px);
}
.playing-card.hovered.directionRight {
  transform: translateX(-30px);
}
```